client/src/app/pages/product/product.component.css

```diff
@@ -0,0 +1,8 @@
+.price {
+    color: red;
```

Comment:
@plubgoard we have a variable for this, you can access it by doing "color: var(--color-text-highlight);"